core/services/vrf/delegate.go

```diff
@@ -120,7 +120,7 @@ func (d *Delegate) ServicesForSpec(jb job.Job) ([]job.Service, error) {
 		gethks:             d.ks.Eth(),
 		pipelineORM:        d.porm,
 		job:                jb,
-		reqLogs:            utils.NewMailbox(1000),
+		reqLogs:            utils.NewMailbox(100_000),
```

Comment:
Use https://github.com/smartcontractkit/chainlink/blob/0a39ed240d40f5025506113d10b0032da4f673fe/core/utils/mailbox.go#L23-L25 instead

Reply:
Agreed, though 0.10.15 doesn't have this method. Anywho, this PR is stale, I should've branched off master to create a hotfix.